### ISSUE TITLE
DataSet Missing

### ISSUE THREAD

**Issue by DevelopedByAnurag** [TheAILearner/Snake-Game-Using-Tensorflow-Object-Detection-API#1]
Bro Even I am also working on a similar project and was in search of some more dataset rather than only my hand dataset so it might be really great if you could share this dataset ??

**Comment by masouduut94** [comment]
@DevelopedByAnurag Do you know any dataset for **gesture recognition** which has images with bounding boxes?

If you found any, please share it with me. I really need one.

**Comment by DevelopedByAnurag** [comment]
@masouduut94  Yes I have Created my own DataSet worth > 10 thousand Images all labeled and contains nearly 16 diffrent gesture each with a diffrent label.. share your mail Id or any other way as it worth nearly 1.5 GB even after being commpressed to lowest.

**Comment by masouduut94** [comment]
@DevelopedByAnurag You are a life saver. This is the my mail masouduut94[at]gmai.[dot]com.  If possible share the link with code and pretrained model. that would save me a lot of time my friend. thanks a million.

**Comment by masouduut94** [comment]
@DevelopedByAnurag  You are a life saver bro :heart_eyes:. This is the my mail masouduut94[at]gmail.[dot]com. If you also had the code and model for the gesture recognition project, it would be nice of you to share it with me. that would save me a lot of time. 

thanks a million.
Best wishes

**Comment by DevelopedByAnurag** [comment]
Okay Its better to upload a that trained model On gthub or bucket and i will mail you back as I have a trained model on RNN inception and SSD mobinet on 22,000 and 56,000 steps respectively.

**Comment by masouduut94** [comment]
@DevelopedByAnurag Did you successfully upload the dataset you mentioned?
And also I wonder if you could contact me, so I can talk to you more comfortably.

My email: masouduut94@gmail.com
